### ISSUE TITLE
Countdown timer instead of circle visualization

### ISSUE THREAD

**Issue by kfeeeeee** [codito/gnome-pomodoro#371]
Unlike shown in the pictures on the web page, running on Ubuntu 18.04, during a pomodoro the timer is visualized as a circle-ish thing.

Is it possible to change the circle in to the normal timer within the top bar?

**Comment by akontsevich** [comment]
I like circle :) Think better to add as an option if necessary.

**Comment by Nepoh** [comment]
Yeah... actually I was looking for a timer in the top bar. I saw the screenshot, installed the app and now there is no timer :-(

**Comment by pooriamo** [comment]
I like the timer :((
At first I thought the circle is a bug!

**Comment by sojusnik** [comment]
You actually can change the indicator by writing the following in the terminal:
```
gsettings set org.gnome.pomodoro.plugins.gnome indicator-type text
gsettings set org.gnome.pomodoro.plugins.gnome indicator-type short-text
gsettings set org.gnome.pomodoro.plugins.gnome indicator-type icon
```
or use `dconf` and search for `indicator-type`.

Here's a similar [request](https://github.com/codito/gnome-pomodoro/issues/262).

**Comment by pooriamo** [comment]
@sojusnik It doesn't work for me, I tried this, dconf, ... and non of them works and I'm still seeing this circle.

**Comment by sojusnik** [comment]
Do you have v0.14.0 installed?

**Comment by pooriamo** [comment]
@sojusnik It's 0.13.4

**Comment by sojusnik** [comment]
Why not install the [latest release](http://gnomepomodoro.org/#download) and try again?

**Comment by pooriamo** [comment]
@sojusnik After I upgraded it, It's not showing at the top bar at all.

**Comment by pooriamo** [comment]
@sojusnik I reset the gsettings and the top icons are back, Removed the old version and installed the new one and it's now working. thanks

**Comment by denkasyanov** [comment]
With version 0.15.1 installed using apt changing settings with 
`gsettings set org.gnome.pomodoro.plugins.gnome indicator-type text` and 
`gsettings set org.gnome.pomodoro.plugins.gnome indicator-type short-text` had no effect.

![image](https://user-images.githubusercontent.com/34678129/70870910-5047b700-1faa-11ea-89a6-617d4a0bdbda.png)

**Comment by pockata-kt** [comment]
I seem to be having the same issue. Version 0.15.1, tried reinstalling, gsettings seem to not change the indicator-type.
I'm on ubuntu 19.10.

**Comment by peterjaap** [comment]
Running Ubuntu 18.04 LTS, installed through apt.

Tried changing settings through gsettings. The setting is updated;

```
$ gsettings set org.gnome.pomodoro.plugins.gnome indicator-type short-text
$ gsettings get org.gnome.pomodoro.plugins.gnome indicator-type          
'short-text'
```

But no effect.

Please fix this. The countdown is the whole reason I installed this Pomodoro timer.

**Comment by attebjorner** [comment]
Ubuntu 20.04 (but the version of the timer is for Ubuntu 19.10):

with `$ gsettings set org.gnome.pomodoro.plugins.gnome indicator-type short-text` it shows only minutes
![Screenshot from 2020-09-20 18-42-55](https://user-images.githubusercontent.com/55637215/93715392-abc65700-fb71-11ea-893e-4771b0476b9b.png)

with `$ gsettings set org.gnome.pomodoro.plugins.gnome indicator-type text` it shows minutes and seconds, as on the screenshot from the homepage
![Screenshot from 2020-09-20 18-42-26](https://user-images.githubusercontent.com/55637215/93715403-bed92700-fb71-11ea-84fc-ed35d0018e64.png)

**Comment by kamilprusko** [comment]
It got improved in the latest app / [extension](https://github.com/focustimerhq/gnome-shell-extension-focus-timer)

<img width="629" height="389" alt="Image" src="https://github.com/user-attachments/assets/d3de1b5b-f182-49f8-911d-2a561033c974" />

<img width="1074" height="626" alt="Image" src="https://github.com/user-attachments/assets/77957fc7-7166-43f6-9d93-824be31b82c7" />
